core/android_validation_constants_test.py

```diff
@@ -75,4 +75,10 @@ def test_exploration_constants_in_both_files_are_equal(self) -> None:
             android_validation_constants.VALID_RTE_COMPONENTS)
         self.assertItemsEqual(  # type: ignore[no-untyped-call]
             language_ids_in_constants,
-            android_validation_constants.SUPPORTED_LANGUAGES)
+            android_validation_constants.SUPPORTED_LANGUAGE_CODES)
+        self.assertItemsEqual(  # type: ignore[no-untyped-call]
+            language_ids_in_constants,
+            list(android_validation_constants.LANGUAGE_CODE_TO_ENUM_MAP.keys()))
+        self.assertItemsEqual(  # type: ignore[no-untyped-call]
```

Comment:
Technically this last check is unnecessary, right? I.e. if you've checked A == B and A == C then it's definitely going to be the case that B == C ...

Reply:
being too cautious here.
fixed now.